plugins/include/msgnatives.inc

```diff
@@ -0,0 +1,206 @@
+ *                        HideElement_Cross_Ammo_WPNList
+ *                        HideElement_Flashlight (+)
+ *                        HideElement_All
+ *                        HideElement_Radar_Health_Armor (+)
+ *                        HideElement_Timer (+)
+ *                        HideElement_Money (+)
+ *                        HideElement_Crosshair
+ * @param noadd         If set to false and the chosen element names have
+ *                      a "+" sign, an additional crosshair will be drawn.
+ * @param reliable      If true, the message will be sent via the reliable
+ *                      channel, otherwise it will use the unreliable one
+ *
+ * @noreturn
+ * @error               If "id" is non-zero and the client isn't
+ *                      connected, an error will be thrown.
+ */
+native hide_hud_elements(id, HideElemenentFlags:elements, bool:noadd = true, bool:reliable = true)
+
+/**
+ * Fades the client's screen.
+ *
+ * @param id            Client index or 0 for all clients
+ * @param duration      How long (in seconds) the fade is going to stay
+ *                      on the screen (0 - 16)
+ * @param fadetime      How many seconds is the fade going to fade in (0 - 16)
+ * @param flags         Screen fade flags:
+ *                        ScreenFade_FadeIn       - default
+ *                        ScreenFade_FadeOut      - fade out (not in)
+ *                        ScreenFade_Modulate     - modulate (don't blend)
+ *                        ScreenFade_StayOut      - ignores the duration and stays faded out until a new ScreenFade messages is received
+ * @param r             Red color amount (0 - 255)
+ * @param g             Green color amount (0 - 255)
+ * @param b             Blue color amount (0 - 255)
+ * @param a             Color alpha (brightness) (0 - 255)
+ * @param reliable      If true, the message will be sent via the reliable
+ *                      channel, otherwise it will use the unreliable one
+ *
+ * @noreturn
+ * @error               If "id" is non-zero and the client isn't
+ *                      connected, an error will be thrown.
+ */
+
+native fade_user_screen(id, Float:duration = 1.0, Float:fadetime = 0.5, ScreenFadeFlags:flags = ScreenFade_FadeIn, r = 0, g = 0, b = 255, a = 75, bool:reliable = true)
+
+/**
+ * Shakes the client's screen.
+ *
+ * @param id            Client index or 0 for all clients
+ * @param amplitude     Shake amplitude (0 - 16)
+ * @param duration      Shake duration (in seconds) (0 - 16)
+ * @param frequency     Delay between each shake (0 - 16)
+ * @param reliable      If true, the message will be sent via the reliable
+ *                      channel, otherwise it will use the unreliable one
+ *
+ * @noreturn
+ * @error               If "id" is non-zero and the client isn't
+ *                      connected, an error will be thrown.
+ */
+native shake_user_screen(id, Float:amplitude = 3.0, Float:duration = 3.0, Float:frequency = 1.0, bool:reliable = true)
```

Comment:
What is the reason for these default values?

Reply:
@WPMGPRoSToTeMa It produces a simple and short shake, setting them to 1.0 is barely noticeable.